src/Nethermind/Nethermind.Verkle.Tree/VerkleTree.Prover.cs

```diff
@@ -63,6 +63,32 @@ public ExecutionWitness GenerateExecutionWitness(byte[][] keys, out Banderwagon
         return new ExecutionWitness(stemStateDiffList, proof);
     }
 
+    public void UpdateExecutionWitness(ExecutionWitness executionWitness, byte[][] keys)
```

Comment:
rename to a more appropiate name - like UpdateWithPostStateValues

Reply:
also no need to pass keys here - you already have that information in executionWitness